controlpanel/api/cluster.py

```diff
@@ -176,8 +176,12 @@ def get_repositories(user):
     repos = []
     github = Github(user.github_api_token)
     for name in settings.GITHUB_ORGS:
-        org = github.get_organization(name)
-        repos.extend(org.get_repos())
+        try:
+            org = github.get_organization(name)
+            repos.extend(org.get_repos())
+        except GithubException as err:
+            log.warning(f'Failed getting {name} Github org repos for {user}: {err}')
+            raise err
```

Comment:
So we're happy with not wrapping this GH exception into another exception in this case 👍

Reply:
There's nothing we can do to handle this exception, so wrapping it is pointless.